### PR TITLE
solution

### PR DESCRIPTION
**Pull request by YukihiroSM**

### CODE REVIEW

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,56 @@
...
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
```
Comment (YukihiroSM) [inline app/create_file.py:16]: you can forget about using index variable here if you will move filename variable creation before break statement. in this case you will be able to use variable, which is already created in line 11 (during using for statement)

**Review by YukihiroSM — COMMENTED** [review]
Please, take a look into comments and fix all what you need. If you have any other questions - please, ping me. Also, test your code for all cases. Try to find edge cases. You can use a piece of paper to think, what user can do with your script. Users are unpredictable, so they can try to do anything :D
Also, I can't see case, when you are checking for the file existence and appending data to the file if it exists. Please, read the task and add this functionality.  (Can't request changes, because I'm a PR owner)
